templates/common/_base/files/clean-ovs-network.yaml

```diff
@@ -0,0 +1,60 @@
+mode: 0755
+path: "/usr/local/bin/clean-ovs.sh"
+contents:
+  inline: |
+    #!/bin/bash
+    set -eux
+
+    # Revert changes made by /usr/local/bin/configure-ovs.sh.
+    # Remove OVS bridge "br-ex". Use the default NIC for cluster network.
+
+    iface=""
+    counter=0
+    # find default gateway interface
+    while [ $counter -lt 12 ]; do
+      # check ipv4
+      iface=$(ip route show default | awk '{if ($4 == "dev") print $5; exit}')
+      if [[ -n "$iface" ]]; then
+        echo "IPv4 Default gateway interface found: ${iface}"
+        break
+      fi
+      # check ipv6
+      iface=$(ip -6 route show default | awk '{if ($4 == "dev") print $5; exit}')
+      if [[ -n "$iface" ]]; then
+        echo "IPv6 Default gateway interface found: ${iface}"
+        break
+      fi
+      counter=$((counter+1))
+      echo "No default route found on attempt: ${counter}"
+      sleep 5
+    done
+
+    if [ "$iface" != "br-ex" ] && ! nmcli connection show br-ex ; then
+      echo "br-ex not exist!"
+      exit 0
+    fi
+
+    # find the default NIC attached bridge
+    if nmcli connection show ovs-port-phys0 &> /dev/null; then
+      iface=$(nmcli c|grep ovs-if-phys0|awk '{print $4}')
+      if [ -z "$iface" ]; then
+        echo "ERROR: Unable to find default gateway interface"
+        exit 1
+      fi
+      nmcli c del ovs-port-phys0 
+    fi
+
+    if nmcli connection show ovs-if-phys0 &> /dev/null; then
+      nmcli c del ovs-if-phys0
+    fi
+
+    if nmcli connection show ovs-port-br-ex &> /dev/null; then
+      nmcli c del ovs-port-br-ex
+    fi
+
+    if nmcli connection show ovs-if-br-ex &> /dev/null; then
+      nmcli c del ovs-if-br-ex
+    fi
+
+    nmcli c del br-ex
+    nmcli device connect $iface
```

Comment:
is this part necessary? The other connections that are lower priority should automatically become active with iface after ovs-if-phys0 gets deleted.

Reply:
fixed.